picard/formats/id3.py

```diff
@@ -684,6 +727,31 @@ def format_specific_metadata(self, metadata, tag, settings=None):
 
         return values
 
+    def _parse_sylt_text(self, text):
+        lrc_lyrics = []
+        previous_line_ended = True
+        for lyrics, milliseconds in text:
+            minutes = milliseconds // (60 * 1000)
+            seconds = (milliseconds % (60 * 1000)) // 1000
+            hundredths = (milliseconds % 1000) // 10
+            if previous_line_ended:
+                lrc_lyrics.append(f"[{minutes:02d}:{seconds:02d}.{hundredths:02d}]{lyrics}")
+                previous_line_ended = False
+            else:
+                lrc_lyrics.append(f"<{minutes:02d}:{seconds:02d}.{hundredths:02d}>{lyrics}")
+            if '\n' in lyrics:
+                previous_line_ended = True
+        return "".join(lrc_lyrics)
+
+    def _parse_lrc_text(self, text):
+        sylt_lyrics = []
+        timestamp_and_lyrics = batched(self.__lrc_syllable_re_parse.split(text)[1:], 2)
```

Comment:
I followed your approach, but I have a question about this:
>4. If A2 Enhanced LRC, then if there is a phrase-timestamp at the very end of the line i.e. followed by nothing or white space, then remove that end-of-line phrase-timestamp.

When working with standard LRC files, I noticed that sometimes there would be line-timestamps with no text after. I think this is done to "clear the screen" when following along with the music. Maybe a phrase-timestamp and the end of a line could be used for the same purpose.

 Also, there could be some phrase-timestamps with no text in the middle of a line, like:
 ```
 <00:10.000>example<00:20.000><00:30.000>example
 ```
I think the expected behaviour would be to create an empty SYLT frame for those, or am I misinterpreting it?

Reply:
I think you may be right - and I think you are asking whether there is a difference between:
```
[00:10.000] some stuff
[00:15.000]
[00:20.000] some stuff
```
where as you suggest I agree that we need to preserve the empty time stamp, and
```
[00:10.000] <00:10.000> some stuff <00:15.000>
[00:20.000] <00:20.000> some stuff
```
which is effectively the same and
```
[00:10.000] <00:10.000> some stuff <00:20.000>
[00:20.000] <00:20.000> some stuff
```
where (because the timestamps are the same) is a null subframe that should not be saved.

So when for A2 Enhanced versions you remove the line-timestamps, then if you get the exact same timestamp repeating (hopefully only ever getting this with nothing in between) then you skip it.

Reply:
@twodoorcoupe, @Sophist-UK  Is there anything open regarding the above?

Reply:
No, I made it so that the line timestamps are removed if there are also syllable timestamps in the text. Then SYLT frames with no lyrics and with a timestamp equal to the successive are removed.